canonical-websites/mir-server.io
# Added the global nav

### Done

- Added the global nav to the site

### QA 

1. grab this branch and ./run it
2. look at http://0.0.0.0:8028/
3. see that the global nav looks good and works

### Issue

Fixes #30

### Screenshot

![image](https://user-images.githubusercontent.com/441217/46619386-e3b96880-cb19-11e8-8b44-88ce6f8e2048.png)
